mobileClient/angularMobileApp/tsconfig.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
this file looks like the tsconfig files seen in zlux-app-manager/virtual-desktop/plugin-config
shouldnt it be moved there so that people who want to build mobile apps can extend a default configuration?